component.json

```diff
@@ -103,6 +103,41 @@
           "required": true,
           "model": "objectTypes",
           "prompt": "Please select a Salesforce Object"
+        },
+        "startTime": {
+          "viewClass": "TextFieldView",
+          "label": "Start Time",
+          "required": false,
+          "placeholder":"2018-11-12T13:06:01.179Z",
+          "note": "Indicates the beginning time to start polling from, default 1970-01-01T00:00:00.000Z"
+        },
+        "endTime": {
+          "viewClass": "TextFieldView",
+          "label": "End Time",
+          "required": false,
+          "placeholder": "2018-11-12T13:06:01.179Z",
+          "note": "If provided, do not fetch records modified after this time (defaults to never)"
+        },
+        "sizeOfPollingPage": {
+          "label": "Size of Polling Page",
+          "viewClass": "SelectView",
+          "required" : false,
+          "model": {
```

Comment:
Perhaps allow input of any integer value?

Reply:
done